ui/src/SpaceDashboard/SpaceDashboard.test.tsx

```diff
@@ -103,10 +104,6 @@ describe('SpaceDashboard', () => {
             await createTestComponent();
         });
 
-        afterEach(() => {
-            jest.clearAllMocks();
-        });
-
```

Comment:
Should the clearAllMocks to into the beforeEach? (Is it really not needed anymore?)

Reply:
I don't think having these in every test file are necessary and if you clear all mocks, all the default mocks set in the __mocks__ files get wiped out and you have to reset those in every file. I didn't see any use for this.